client/my-sites/subscribers/components/subscriber-list-container/subscriber-list-container.tsx

```diff
@@ -50,11 +50,11 @@ const SubscriberListContainer = ( {
 		if ( ! isLoading && subscribers.length === 0 && page > 1 ) {
 			pageChangeCallback( pages ?? 0 );
 		}
-	}, [ isLoading, subscribers, page, pageChangeCallback ] );
+	}, [ isLoading, subscribers, page, pageChangeCallback, pages ] );
 
 	return (
 		<section className="subscriber-list-container">
-			{ ! isLoading && ( Boolean( grandTotal ) || searchTerm ) && (
+			{ Boolean( grandTotal ) && (
```

Comment:
I wonder if we could leave this as `{ grandTotal && (` 🤔

Reply:
Yes, indeed. I'll do it :)